### DISCUSSION TITLE
Alfen setup?

### DISCUSSION THREAD

**Discussion by bobf41** [lbbrhzn/ocpp#115] [Q&A] [answered]
I have OCPP setup and configured my connected Alfen EVE Mini S Line like ws://homeassistant.local:9000 but get no info. Same for http:// 
homeassistant.local is retrieved successfully and tried the ipaddress as well.

**Comment by lbbrhzn** [comment, marked as answer]
@Rep-hueman has tested OCPP with Alfen chargers, (Alfen - Eve Single S-line and Pro-line). See issue #19.

@bobf41, can you make a screenshot of how you configured your charger?

**Reply by bobf41** [reply]
Ok, just removed the OCPP install and repo and reïnstalled it.
It started receiving messages now.
No temperature and voltage yet but it does show the Firmware versionnumber, Status etc.

**Reply by lbbrhzn** [reply]
Note that each charger model supports a different subset of measurands, so even though you configure a measurand in OCPP, it may not be sent by your charger.

**Reply by lbbrhzn** [reply]
@bobf41, can you share a screenshot of the model and firmware number? Just to compare against the info shared by @Rep-Hueman.

**Reply by bobf41** [reply]
Model: Eve Mini (S-Line)
Plartform: NG910
Hardware version controller board: G-00
Hardware version power board: F-FEDAULT
Software version controller board: 4.15.0-3910
NFC Reader 1 hardw. version 217H

**Reply by Rep-Hueman** [reply]
These are the supported Subset of measurands

"Energy.Active.Import.Register"
"Current.Import"
"Current.Offered"
"Frequency"
"Power.Active.Import"
"Power.Factor"
"Temperature"
"Voltage"

Sometimes you need to restart the charger.
And then wait 10 minutes and then start the first charging session.
If it's good you see all the value

![Alfen-select](https://user-images.githubusercontent.com/28707705/127770019-7220f414-a5fc-42ed-b2cc-06f8d02ac56f.jpg)
![Alfen-select2](https://user-images.githubusercontent.com/28707705/127770023-a081e6c2-d487-4447-a38c-f7f99f658daf.jpg)

**Reply by bobf41** [reply]
Thanks I will try connecting a car. 
any idea if it will be possible to connect to an official Backoffice and to Homeautomation at the same time? I guess not since it is set through the same parameters

**Reply by lbbrhzn** [reply]
> Any idea if it will be possible to connect to an official Backoffice and to Homeautomation at the same time? I guess not since it is set through the same parameters

I don't know how it works for Alfen chargers, but my Wallbox Pulsar always connects to the Wallbox Backoffice, even when a local ocpp server is configured.

**Reply by keesvanbemmel** [reply]
> These are the supported Subset of measurands
> 
> "Energy.Active.Import.Register" "Current.Import" "Current.Offered" "Frequency" "Power.Active.Import" "Power.Factor" "Temperature" "Voltage"


How did you determine the lis of supported measurands?

**Comment by lbbrhzn** [comment]
> Any idea if it will be possible to connect to an official Backoffice and to Homeautomation at the same time? I guess not since it is set through the same parameters

I don't know how it works for Alfen chargers, but my Wallbox Pulsar always connects to the Wallbox Backoffice, even when a local ocpp server is configured.

**Comment by Ascotg** [comment]
Some extra screenshots of the setup on ACE Service installer that may help people with Alfen Chargers
![image](https://github.com/lbbrhzn/ocpp/assets/29706378/1cc626c2-17cd-4c7d-9f79-41c5e6ec878f)
![image](https://github.com/lbbrhzn/ocpp/assets/29706378/27d13232-65ac-470b-a713-51c326472dbc)

**Reply by bobf41** [reply]
Will we be able to have a Backoffice configured and have a config for HA?
The Backoffice is most important.

**Reply by Ascotg** [reply]
@bobf41 From what I learned, not using the OCPP implementation. An EV charger can only connect to 1 backoffice at a time, which makes sense. The backoffice (OCPP) is meant for the financial side of the charger.

However, you can interface with the Alfen over Modbus TCP protocol and set a backoffice at the same time. Here's a link to someone with that setup: https://community.home-assistant.io/t/alfen-eve-pro-ev-chargepoint-interface-via-tcp-modbus/346000

**Reply by MathyV** [reply]
@Ascotg how did you get the ACE license? Are you an official installer or can anyone request them?

**Reply by MathyV** [reply]
I managed to get the login by just registering on their service portal, if anyone else reads this in the future.
